src/main/java/edu/ucla/library/bucketeer/JobFactory.java

```diff
@@ -272,8 +273,8 @@ private boolean hasHeaderErrors(final String[] aHeaders, final String aJobName)
         int itemIdCount = 0;
         int viewingHintCount = 0;
 
-        for (int columnIndex = 0; columnIndex < aHeaders.length; columnIndex++) {
-            switch (aHeaders[columnIndex]) {
+        for (final String aHeader : aHeaders) {
```

Comment:
I see you decided to follow Codacy's suggestion :)

Reply:
Heh, I think Eclipse actually auto-corrected this for me based on a preference to use this `for` syntax when the index isn't used elsewhere in the loop. I could be wrong about this, but think when I looked at Codacy's suggestion you were using the index in the `for` loop in another place other than just in the `switch`. At the time, given that, it didn't make sense to me to use this syntax. It might just be that I just saw it in the `switch`, though, and Eclipse was smarter than me, and realized it wasn't really needed there.  :laughing: